src/views/CharacterBuild.js

```diff
@@ -22,12 +46,41 @@ export default function CharacterBuild() {
     <>
       {map(keys(charBuild), (stats, idx) => {
         return (
-          <Box key={idx} style={{ display: "flex" }}>
-            <Box style={{ justifyContent: "flex-start", minWidth: "250px" }}>
+          <Box
+            key={idx}
+            style={{ display: "flex", alignItems: "center" }}
+            marginBottom="20px"
+          >
+            <Box
+              style={{
+                justifyContent: "flex-start",
+                minWidth: "250px",
+                textAlign: "right"
+              }}
+              marginRight="20px"
+            >
               {stats}
             </Box>
             <Box style={{ justifyContent: "flex-end", minWidth: "100px" }}>
-              {get(charBuild, stats)}
+              <TextField
+                type={stats === "character_name" ? "text" : "number"}
+                label={stats}
+                variant="outlined"
+                inputProps={{
+                  style: {
+                    color: "white"
+                  }
+                }}
+                size="small"
+                className={classes.field}
+                onChange={e => {
+                  if (stats === "character_name") {
```

Comment:
Character name should not be allowed to be changed. Use _.omit to remove it from the stat list

Reply:
do we just use a static character name for now then? or a drop down menu of the available ones

Reply:
once we complete the actual calc, then we can worry about adding other characters and tailoring to them